templates/cf-apache.service.mustache

```diff
@@ -17,4 +17,3 @@ UMask=0177
 
 [Install]
 WantedBy=multi-user.target
-WantedBy=cfengine3.service
```

Comment:
Could put link to related PR / commit in commit message.

Changelog entry does not explain the result of this change to a user.

Reply:
changed.